app/views/layouts/application.html.slim

```diff
@@ -1,8 +1,7 @@
 doctype html
 html
   head
-    title
-      | HeroesTeamApp
+    = display_meta_tags site: 'HeroesTeamApp'
```

Comment:
rename to "Heroes Team App"

and use "reverse" style ... basically show site name at the end of title